compiler/src/diagnostics/comment_attributes.re

```diff
@@ -0,0 +1,37 @@
+exception InvalidAttribute(string);
```

Comment:
Why did you split this into a different file?

Reply:
There was a circular dependency, parser -> comments -> parser 😢